applications/creating_applications/snippets

```diff
@@ -0,0 +1 @@
+../snippets
```

Comment:
This link should level up to the top-level snippets/ directory (so ../../snippets) instead of inside the applications/ folder's snippets link.

Same for the other snippets links in the PR that are two levels deep if you can change those as well please!